### PR TITLE
feat: display ckBTC to BTC estimated time

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Because the transfer on BTC network is slow, display a static estimation of the time it can least.

# Changes

- display 30min estimated time on review step and in success msg

# Screenshots

Capture d’écran 2023-03-13 à 15 08 20

Capture d’écran 2023-03-13 à 15 08 25

### CODE REVIEW

File: frontend/src/lib/modals/accounts/NewTransaction/TransactionReview.svelte [hunk 1]
```diff
@@ -79,6 +83,15 @@
     </div>
 
     <slot name="additional-info" />
+
+    {#if networkBtc}
```
Comment (lmuntaner) [inline frontend/src/lib/modals/accounts/NewTransaction/TransactionReview.svelte:87]: Why not adding it in the `additional-info` instead?

This way the `TransactionReview` doesn't know about this specifics of ckbtc

**Review by lmuntaner — COMMENTED** [review]
A question

File: frontend/src/lib/modals/accounts/NewTransaction/TransactionReview.svelte (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/modals/accounts/NewTransaction/TransactionReview.svelte:87]: Sure why not, will try

Comment (peterpeterparker) [reply frontend/src/lib/modals/accounts/NewTransaction/TransactionReview.svelte]: Done, thanks

**Review by lmuntaner — APPROVED** [review]
Thanks!

**Pull request merged**